dts/bindings/dac/ti,dacx0508-base.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2020 M2I Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+include: [dac-controller.yaml, spi-device.yaml]
+
+properties:
+    voltage-reference:
+      type: int
+      required: true
+      description: DAC voltage reference select
+
+    channel0-gain:
+      type: int
+      required: true
+      description: Channel 0 gain select
+
+    channel1-gain:
+      type: int
+      required: true
+      description: Channel 1 gain select
+
+    channel2-gain:
+      type: int
+      required: true
+      description: Channel 2 gain select
+
+    channel3-gain:
+      type: int
+      required: true
+      description: Channel 3 gain select
+
+    channel4-gain:
+      type: int
+      required: true
+      description: Channel 4 gain select
+
+    channel5-gain:
+      type: int
+      required: true
+      description: Channel 5 gain select
+
+    channel6-gain:
+      type: int
+      required: true
+      description: Channel 6 gain select
+
+    channel7-gain:
+      type: int
+      required: true
+      description: Channel 7 gain select
+
+    "#io-channel-cells":
+      const: 1
+
+io-channel-cells:
+    - output
```

Comment:
As a permanent solution I would prefer adding a new cell `gain` to `io-channel-cells`. But that would require a change in the API to pass it to the driver. So this is OK as a preliminary workaround.